host/hackrf-tools/src/hackrf_sweep.c

```diff
@@ -266,6 +274,14 @@ int rx_callback(hackrf_transfer* transfer)
 					}
 				}
 				sweep_count++;
+
+				if (timestamp_normalized == true)
+				{
+					// set the timestamp of the next sweep
+					memset(&usb_transfer_time, 0, sizeof (usb_transfer_time));
```

Comment:
I don't think there's any need for this `memset` call - the structure is immediately overwritten by the following `gettimeofday` call.